tests/system/action/assignment/test_create.py

```diff
@@ -131,20 +139,61 @@ def test_create_full_fields(self) -> None:
                 "phase": "search",
                 "default_poll_description": "text_test2",
                 "number_poll_candidates": True,
+                "attachment_ids": [1],
```

Comment:
please rename attachment_ids to mediafile_ids to reflect the collection name in the field name

Reply:
Done something like that